### PR TITLE
.ci/setup_env_centos.sh: Fixes and improvements

### LINKED ISSUE #2214: Enable PowerTools repository for CentOS 8
We need to enable PowerTools repository for CentOS 8 in order to be
able to install packages like moreutils, libseccomp-devel, pandoc, etc.

### PR DESCRIPTION
**Pull request by wainersm**
On CentOS 8 the script is going to configure PowerTools repository
by using the yum-config-manager command (see commit 333de7c6), that
might not be installed in the system. This change ensures the package
yum-utils (yum-config-manager provider) is present.

Fixes #2214

Signed-off-by: Wainer dos Santos Moschetta <wainersm@redhat.com>

### CODE REVIEW

File: .ci/setup_env_centos.sh [hunk 1]
```diff
@@ -42,6 +42,7 @@ sudo -E yum -y update
 
 if [ "$centos_version" == "8" ]; then
 	echo "Enable PowerTools repository"
+	sudo -E yum -y yum-utils
```
Comment (wainersm) [inline .ci/setup_env_centos.sh:45]: Missed `install`... let me send an update...

**Comment by wainersm** [comment]
@GabyCT Hi! I've two other patches for this same script. Should I send separate PRs or can I just append on this one?

**Comment by GabyCT** [comment]
@wainersm you can append it :)

**Comment by wainersm** [comment]
Hi @GabyCT , probably I asked you before (and forgot): what should I do when the PR is approved and passed CI? I merge it myself or I wait someone else to do it.

**Review by GabyCT — APPROVED** [review]

**Pull request merged**